### PR TITLE
Feature/212 introduce tenancy to phishtray

### PR DESCRIPTION
**Pull request by celopacob**
Tenancy based on organizations. Had to implement a new custom User object in order to associate it to organizations.
Never did this from scratch before, so needs a thorough review :-)

### CODE REVIEW

**Comment by DanielGeoffreyHodgkins** [comment]
might be worth merging master into this branch or rebasing!

File: users/admin.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+from .forms import CustomUserCreationForm, CustomUserChangeForm
+from .models import User
+
+
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    add_form = CustomUserCreationForm
+    form = CustomUserChangeForm
+    list_display = ['id', 'email', 'username', 'first_name', 'last_name']
```
Comment (DanielGeoffreyHodgkins) [inline users/admin.py]: is this formatted with black? We don't have the pre-commit hook in phishtray so you might have to set it up on PyCharm or whatever IDE you use! https://black.readthedocs.io/en/stable/editor_integration.html

Comment (celopacob) [reply users/admin.py]: Probably!! Thanks

File: participant/queryset.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from django.db.models import QuerySet, Q
```
[the remark this answers is not in the document]
Comment (celopacob) [reply participant/queryset.py]: Done!

**Pull request merged**